applications/workers/release/combined-worker/src/main/kotlin/net/corda/applications/workers/combined/CombinedWorker.kt

```diff
@@ -211,6 +208,43 @@ class CombinedWorker @Activate constructor(
         schedulerProcessor.start(config)
     }
 
+    private fun createExtraConfigs(params: CombinedWorkerParams): List<Config> {
+        val extraConfigs = mutableListOf(
+            createConfigFromParams(BootConfig.BOOT_CRYPTO, createCryptoBootstrapParamsMap(params.hsmId)),
+            createConfigFromParams(BootConfig.BOOT_REST, params.restParams),
+            createConfigFromParams(BOOT_WORKER_SERVICE, params.workerEndpoints),
+        )
+
+        if(params.mediatorReplicasFlowSession != null) {
```

Comment:
Nit:
```suggestion
        if (params.mediatorReplicasFlowSession != null) {
```
and `if` statements below.